manageiq-ui-classic.gemspec

```diff
@@ -20,5 +20,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+
+  s.add_development_dependency "codeclimate-test-reporter", "~> 1.0.0"
```

Comment:
I had some issues with uploading the coverage results when this was here. But I don't see it here ... weird :confused: